javav2/example_code/athena/src/main/java/aws/example/athena/StartQueryExample.java

```diff
@@ -1,175 +1,153 @@
+//snippet-keyword:[AWS SDK for Java v2]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Athena]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[11/02/2020]
+//snippet-sourceauthor:[scmacdon - aws]
+/*
+   Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0
+*/
+
+//snippet-start:[athena.java2.StartQueryExample.complete]
+//snippet-start:[athena.java.StartQueryExample.complete]
+package aws.example.athena;
+
+//snippet-start:[athena.java2.StartQueryExample.import]
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.athena.AthenaClient;
+import software.amazon.awssdk.services.athena.model.QueryExecutionContext;
+import software.amazon.awssdk.services.athena.model.ResultConfiguration;
+import software.amazon.awssdk.services.athena.model.StartQueryExecutionRequest;
+import software.amazon.awssdk.services.athena.model.StartQueryExecutionResponse;
+import software.amazon.awssdk.services.athena.model.AthenaException;
+import software.amazon.awssdk.services.athena.model.GetQueryExecutionRequest;
+import software.amazon.awssdk.services.athena.model.GetQueryExecutionResponse;
+import software.amazon.awssdk.services.athena.model.QueryExecutionState;
+import software.amazon.awssdk.services.athena.model.GetQueryResultsRequest;
+import software.amazon.awssdk.services.athena.model.GetQueryResultsResponse;
+import software.amazon.awssdk.services.athena.model.ColumnInfo;
+import software.amazon.awssdk.services.athena.model.Row;
+import software.amazon.awssdk.services.athena.model.Datum;
+import software.amazon.awssdk.services.athena.paginators.GetQueryResultsIterable;
+import java.util.List;
+//snippet-end:[athena.java2.StartQueryExample.import]
+
+public class StartQueryExample {
+    //snippet-start:[athena.java2.StartQueryExample.main]
+    public static void main(String[] args) throws InterruptedException {
+
+        AthenaClient athenaClient = AthenaClient.builder()
+                .region(Region.US_WEST_2)
+                .build();
+
+        String queryExecutionId = submitAthenaQuery(athenaClient);
+        waitForQueryToComplete(athenaClient, queryExecutionId);
+        processResultRows(athenaClient, queryExecutionId);
+        athenaClient.close();
+    }
+
+    /**
+     * Submits a sample query to Athena and returns the execution ID of the query.
+     */
+    public static String submitAthenaQuery(AthenaClient athenaClient) {
+
+        try {
+
+            // The QueryExecutionContext allows us to set the Database.
+            QueryExecutionContext queryExecutionContext = QueryExecutionContext.builder()
+                .database(ExampleConstants.ATHENA_DEFAULT_DATABASE).build();
+
+            // The result configuration specifies where the results of the query should go in S3 and encryption options

```

Comment:
Amazon Simple Storage Service (Amazon S3) on first usage, Amazon S3 thereafter (everywhere).

Reply:
Fixed that